src/components/standalone/multi-wan/PolicyCreator.vue

```diff
@@ -0,0 +1,405 @@
+              initial_state: mwanDefaults.data!.values.initial_state,
+              family: mwanDefaults.data!.values.protocol,
+              track_ip: mwanDefaults.data!.values.track_ip,
+              track_method: mwanDefaults.data!.values.tracking_method,
+              reliability: mwanDefaults.data!.values.tracking_reliability,
+              count: mwanDefaults.data!.values.ping_count,
+              size: mwanDefaults.data!.values.ping_size,
+              max_ttl: mwanDefaults.data!.values.ping_max_ttl,
+              timeout: mwanDefaults.data!.values.ping_timeout,
+              interval: mwanDefaults.data!.values.ping_interval,
+              failure_interval: mwanDefaults.data!.values.ping_failure_interval,
+              recovery_interval: mwanDefaults.data!.values.ping_recovery_interval,
+              down: mwanDefaults.data!.values.interface_down_threshold,
+              up: mwanDefaults.data!.values.interface_up_threshold
+            }
+          })
+        )
+    )
+    // member creation
+    calls.push(
+      // for each priority we'll have +10 metric applied
+      ...form.value.priorities
+        .map((priority, index) =>
+          // call 'add' to each gateway
+          priority.map((gateway) =>
+            ubusCall('uci', 'add', {
+              config: 'mwan3',
+              name: `${gateway.id}_M${(index + 1) * 10}_W${gateway.weight ?? 100}`,
+              type: 'member',
+              values: {
+                interface: gateway.id,
+                metric: (index + 1) * 10,
+                weight: gateway.weight ?? 100
+              }
+            })
+          )
+        )
+        // flatten array, we have nested promises here
+        .flat(1)
+    )
+    calls.push(
+      // create policy
+      ubusCall('uci', 'add', {
+        config: 'mwan3',
+        name: form.value.label,
+        type: 'policy',
+        values: {
+          // map gateways to members name
+          // TODO: reuse name generation logic from above
+          use_member: form.value.priorities
+            .map((priority, index) =>
+              priority.map(
+                (gateway) => `${gateway.id}_M${(index + 1) * 10}_W${gateway.weight ?? 100}`
+              )
+            )
+            .flat(1)
+        }
+      })
+    )
+    // if this is the default policy, create a default rule too
+    // TODO: Rule name?
+    if (props.createDefault) {
+      calls.push(
+        ubusCall('uci', 'add', {
+          config: 'mwan3',
+          name: 'All',
+          type: 'rule',
+          values: {
+            proto: 'all',
+            src_ip: '0.0.0.0/0',
+            dest_ip: '0.0.0.0/0',
+            sticky: '0',
+            use_policy: form.value.label
+          }
+        })
+      )
+    }
+    // If all calls are successful, go forth and reload the UI
+    // TODO: implement error catching, with rollback.
+    Promise.all(calls).then(() => {
+      saving.value = false
+      emit('policyCreated')
+    })
```

Comment:
Can we provide a basic error management? An inline notification showing "Cannot create policy" should be enough